insights/parsers/ip.py

```diff
@@ -573,3 +606,92 @@ class Ipv6Neigh(IpNeighParser):
     Class to parse ``ip -6 neigh show nud all`` command output.
     """
     pass
+
+
+@parser("ip-s_link")
+class IpLinkInfo(IpAddr):
```

Comment:
@vishwanathjadhav to be consistent with the doc structure of this module you'll need to add a subtitle in the docstring at the top of this module to reference `IpLinkInfo`.  For instance add a line like:

```
IpLinkInfo - command ``ip -s link``
-----------------------------------
```